eo-runtime/src/main/eo/org/eolang/negative-infinity.eo

```diff
@@ -128,12 +130,9 @@
     x.at 0 > first!
     if. > @
       or.
-        first.as-bytes.eq (nan.as-bytes)
+        not.
+          first.eq first
```

Comment:
@maxonfjvipon I don't clearly understand: Why the next code:
```eo
not.
  first.eq first
```
is equal to 
```eo
first.as-bytes.eq (nan.as-bytes)
```

Reply:
@volodya-lombrozo it's NaN checking. We can say that number is NaN either compare its bytes to NaN bytes, or we can use the rule that NaN != NaN is TRUE

Reply:
@maxonfjvipon Maybe we can leave at least some comment above about such a behaviour? Because it's far from intuitive understanding.